Jellyfin.Api/Auth/PlaybackPolicy/PlaybackHandler.cs

```diff
@@ -37,16 +37,15 @@ public class PlaybackHandler : AuthorizationHandler<PlaybackRequirement>
     /// <inheritdoc />
     protected override Task HandleRequirementAsync(AuthorizationHandlerContext context, PlaybackRequirement requirement)
     {
-        var ip = _httpContextAccessor.HttpContext?.GetNormalizedRemoteIp();
-
-        // Loopback will be on LAN, so we can accept null.
-        if (ip is null || _networkManager.IsInLocalNetwork(ip))
+        var ip = _httpContextAccessor.HttpContext?.GetNormalizedRemoteIP();
+        var userId = context.User.GetUserId();
+        if (userId.Equals(default) && (ip is null || _networkManager.IsInLocalNetwork(ip)))
```

Comment:
I think it's incorrect to succeed the requirement if the userId is empty. Theoretically this only happens if authenticating via api key, but I would like to protect against it